src/tests/features/rich_rules.at

```diff
@@ -319,4 +323,4 @@ IP6TABLES_LIST_RULES([filter], [IN_foobar_post], 0, [dnl
     ACCEPT 0 -- ::/0 ::/0
 ])
 
-FWD_END_TEST([-e '/ERROR: INVALID_ZONE:/d'])
+FWD_END_TEST([-e '/ERROR: INVALID_ZONE:/d' -e "/ERROR: INVALID_SERVICE: Policy 'foobar': 'listen' not among existing services/d"])
```

Comment:
You can avoid the long line by using `dnl`.
```m4
FWD_END_TEST([-e '/ERROR: INVALID_ZONE:/d' dnl
              -e "/ERROR: INVALID_SERVICE: Policy 'foobar': 'listen' not among existing services/d"])
```

Reply:
will change.